api/util/util.go

```diff
@@ -82,3 +88,126 @@ func RemoveDuplicates(s []string) []string {
 	}
 	return s[:i]
 }
+
+// CheckMaliciousInput checks if an user's input is "malicious" or not
+func CheckMaliciousInput(repository types.Repository, c echo.Context) (string, error) {
```

Comment:
`CheckValidInput` can be a better name/description?

Reply:
Absolutely! I have just done that at 38eab85.